man/systemd.special.xml

```diff
@@ -870,6 +870,17 @@
     <filename>printer.target</filename>,
     <filename>smartcard.target</filename>,
     <filename>sound.target</filename>.</para>
+
+    <para>There is also <filename>graphical-session.target</filename> which
+    provides an alias for the currently running graphical session. A session like
```

Comment:
"alias" has a specific meaning in the systemd parlance, basically a second name for a unit. So some different wording has to be found.

Also, I don't think a paragraph should start with "There is also". ;)